pom.xml

```diff
@@ -45,15 +45,16 @@
 
     <junit.version>4.12</junit.version>
     <jmh.version>1.12</jmh.version>
+    <jmock.version>2.6.0</jmock.version>
 
     <k3po.version>3.0.0-alpha-77</k3po.version>
     <k3po.nukleus.ext.version>0.7.1</k3po.nukleus.ext.version>
 
     <nukleus.plugin.version>0.10</nukleus.plugin.version>
     <nukleus.version>0.6.2</nukleus.version>
-
     <nukleus.http2.spec.version>0.16</nukleus.http2.spec.version>
-    <reaktor.test.version>0.10</reaktor.test.version>
+    <reaktor.test.version>0.9.2</reaktor.test.version>
+    <k3po.nukleus.ext.version>0.5</k3po.nukleus.ext.version>
```

Comment:
Please remove this duplicate `k3po.nukleus.ext.version` property and retain version `10` for `reaktor.test.version`.